src/styles/index.css

```diff
@@ -0,0 +1,188 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+  color: rgb(6, 11, 53);
+}
+
+.page__body {
+  margin: 0;
+}
+
+.nav {
+  text-transform: uppercase;
+  font-weight: 500;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__list :not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  text-decoration: none;
+  color: rgb(6, 11, 53);
+  padding-block: 23px;
+  transition: 300ms;
+}
+
+.nav__link--hover:hover {
+  color: rgb(0, 172, 220);
+}
+
+.nav__link--active {
+  color: rgb(0, 172, 220);
+}
+
+.nav__link--active::before {
+  content: '';
+  display: block;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: rgb(0, 172, 220);
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline: 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header__logo-link {
+  display: flex;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars :not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--active :not(:last-child) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars__item {
+  width: 16px;
+  height: 16px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.buy {
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 12px 70px;
+  color: rgb(255, 255, 255);
+  background-color: rgb(0, 172, 220);
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: 5px;
+  transition: 300ms;
+}
+
+.buy:hover {
+  background-color: rgb(255, 255, 255);
+  color: rgb(0, 172, 220);
+  border: 1px solid rgb(0, 172, 220);
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 200px));
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid rgb(243, 243, 243);
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  transition: 300ms;
+}
+
+.card:hover {
+  transform: scale(1.2);
```

Comment:
The 'transform: scale(1.2)' on hover for the '.card' class may cause layout shifts and affect user experience. Consider using a smaller scale factor or other visual effects that do not significantly alter the layout.